### PR TITLE
Create BLorient13268 according to S. Strelcyns catalogue

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient13268.xml [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Oriental</collection>
+                        <idno>BL Oriental 13268</idno>
+                        <altIdentifier><idno>Strelcyn 31</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <msItem xml:id="ms_i1">
+                            <locus from="3r" to="30vb"/>
+                            <title ref="LIT2701Dawit" type="incomplete"/>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient13268.xml:34]: If all are fragments, type="incomplete" could be added to the titles, I think

**Review by thea-m — APPROVED** [review]

File: LondonBritishLibrary/orient/BLorient13268.xml [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+                            <list>
+                                <item xml:id="a1">
+                                    <locus from="1v" to="2v"/>
+                                    <desc type="GuestText">Salām to the Virgin Mary (fragment).</desc>        
+                                </item>
+                            </list>
+                        </additions>
+                        <bindingDesc>
+                            <binding xml:id="binding">
+                                <decoNote xml:id="b1" type="unspecified">No binding.</decoNote>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient13268.xml:107]: I do not think `decoNote `is suitable to describe the absence of anything, I think it has more to do with condition.
I also wonder if the leaves are completely loose (unbound) or if the thing lacking are just the covers (seems more probable to me, that the description is that of a disintegrated manuscript partially fallen apart)
```
<binding>
                                <condition key="deficient">covers absent or lost</condition>
                            </binding>
```
NB for most elements no `xml:id` needed unless you refer to the element from elsewhere

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient13268.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient13268.xml:107]: Strelcyn literally says: "unbound". In another place, he speaks about "original flyleafs".
 I therefore assume that it is a pile of loose sheets.

Comment (eu-genia) [reply LondonBritishLibrary/orient/BLorient13268.xml]: OK then
```
<binding>
                                <condition key="other">unbound</condition>
                            </binding>
```

**Comment by CarstenHoffmannMarburg** [comment]
@eu-genia 

I think, all requests are solved or clarified. Can this branch be merged?

**Review by eu-genia — APPROVED** [review]

**Pull request merged**